plugins/org.eclipse.xtext.ui/META-INF/MANIFEST.MF

```diff
@@ -78,6 +78,7 @@ Export-Package: org.eclipse.xtext.common.ui.contentassist,
  org.eclipse.xtext.ui.editor.toggleComments,
  org.eclipse.xtext.ui.editor.utils,
  org.eclipse.xtext.ui.editor.validation,
+ org.eclipse.xtext.ui.file;x-friends:="org.eclipse.xtext.ui.tests",

```

Comment:
I'm not completely convinced by the package name. Looks more like ui.projectModel or ui.projects